.github/workflows/label-pull-requests.yml

```diff
@@ -0,0 +1,18 @@
+name: Label Pull Requests
+
+on:
+  pull_request:
+    types: [ opened, edited ]
```

Comment:
this seems to be incorrect
https://renato66.github.io/auto-label/

```suggestion
    types: [ 'opened', 'edited' ]
```

Reply:
No, not needed, these are built-in types